zwave_js_server/model/endpoint.py

```diff
@@ -68,6 +67,17 @@ def user_icon(self) -> Optional[int]:
         """Return user icon property."""
         return self.data.get("userIcon")
 
+    def update(
+        self,
+        data: EndpointDataType,
+        values: Optional[Dict[str, Union[ConfigurationValue, Value]]] = None,
+    ) -> None:
+        """Update the endpoint data."""
+        self.data = data
+        for value_id, value in (values or {}).items():
+            if value_id not in self.values:
```

Comment:
If the id is already there, how will the value be updated when state is updated?

Reply:
the value stored on the endpoint is the same instance as the one stored on the node. The node updates the existing values so we just need to make sure our dictionary has all of the values